edenai_apis/apis/tenstorrent/outputs/text/question_answer_output.json

```diff
@@ -0,0 +1,11 @@
+{
+  "original_response": {
```

Comment:
One thing I wanted to clarify is how did you find which example query to make? I forgot.

Reply:
This is the example from the Eden docs for QA.